ftp-edi-message-to-salesforce-opportunity/main.bal

```diff
@@ -0,0 +1,119 @@
+import ballerinax/edifact.d03a.retail.mREQOTE;
+
+import ballerina/io;
+import ballerinax/salesforce as sf;
+import ballerina/file;
+import ballerina/ftp;
+
+configurable ftp:ClientConfiguration ftpConfig = ?;
+configurable string ftpNewQuotesPath = ?;
+configurable string ftpProcessedQuotesPath = ?;
+configurable sf:ConnectionConfig salesforceConfig = ?;
+configurable string salesforcePriceBookId = ?;
+
+public function main() returns error? {
+    sf:Client sfClient = check new (salesforceConfig);
+    ftp:Client ftpClient = check new ftp:Client(ftpConfig);
+    ftp:FileInfo[] quoteList = check ftpClient->list(ftpNewQuotesPath);
```

Comment:
IMO, it will be clear if we add a comment before each major step like fetch data from FTP, update sf, update FTP back etc.